source/renderer/app/containers/wallet/dialogs/wallet-import/WalletFileImportStepContainer.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+import React, { Component } from 'react';
+import { observer, inject } from 'mobx-react';
+import type { InjectedDialogContainerStepProps } from '../../../../types/injectedPropsType';
+import { InjectedDialogContainerStepDefaultProps } from '../../../../types/injectedPropsType';
+import WalletImportFileDialog from '../../../../components/wallet/wallet-import/WalletImportFileDialog';
+import type { ImportFromOption } from '../../../../types/walletExportTypes';
+
+type Props = InjectedDialogContainerStepProps;
+const DefaultProps = InjectedDialogContainerStepDefaultProps;
+
+@inject('stores', 'actions')
+@observer
+export default class WalletFileImportStepContainer extends Component<Props> {
+  static defaultProps = DefaultProps;
+
+  onSelectExportSourcePath = (params: { importFrom: ImportFromOption }) => {
+    this.props.actions.walletMigration.selectExportSourcePath.trigger(params);
+  };
+
+  onResetExportSourcePath = () => {
+    this.props.actions.walletMigration.resetExportSourcePath.trigger();
+  };
+
+  onOpen = () => {
+    this.props.actions.walletMigration.resetMigration.trigger();
+    this.props.actions.walletMigration.initiateMigration.trigger();
```

Comment:
@DeeJayElly please check if this change breaks the automatic migration.
The thing is that the automatic migration should always show only the 2nd step of the migration dialog.